test.js

```diff
@@ -28,6 +32,54 @@ test('.languages', t => {
 	t.true(m.languages.has('de'));
 });
 
+test('Get all English names', testAll, 'en', [
+	'Bulbasaur',
+	'Ivysaur',
+	'Venusaur'
+]);
+
+test('Get all English names (when no language code is given)', testAll, undefined, [
+	'Bulbasaur',
+	'Ivysaur',
+	'Venusaur'
+]);
+
+test('Get all French names', testAll, 'fr', [
+	'Bulbizarre',
+	'Herbizarre',
+	'Florizarre'
+]);
+
+test('Get all German names', testAll, 'de', [
+	'Bisasam',
+	'Bisaknosp',
+	'Bisaflor'
+]);
+
+test('Get all Simplified Chinese names', testAll, 'zh-Hans', [
+	'妙蛙种子',
+	'妙蛙草',
+	'妙蛙花'
+]);
+
+test('Get all Traditional Chinese names', testAll, 'zh-Hant', [
+	'妙蛙種子',
+	'妙蛙草',
+	'妙蛙花'
+]);
+
+test('Get all Japanese names', testAll, 'ja', [
+	'フシギダネ',
+	'フシギソウ',
+	'フシギバナ'
+]);
+
+test('Get all Russian names', testAll, 'ru', [
+	'Бульбазавр',
+	'Айвизавр',
+	'Венузавр'
+]);
```

Comment:
No need to test this for all localization. Testing one localization other than English is enough. That's how it's done with the other tests.